File: langgraph/data/equiv/60.py
def calculate_total_up_to(number: int):
    total_sum = 0
    current = 0
    
    while current <= number:
        total_sum += current
        current += 1
    
    return total_sum

# method name: sum_to_n -> calculate_total_up_to
# parameter name: n -> number
# replaced sum(range(n + 1)) with a while loop to manually accumulate the sum
# introduced two new variables: total_sum and current

File: langgraph/result/qwen/7b-eq2js/60.js
function calculateTotalUpTo(number) {
    let totalSum = 0;
    let current = 0;
    
    while (current <= number) {
        totalSum += current;
        current += 1;
    }
    
    return totalSum;
}

// method name: sumToN -> calculateTotalUpTo
// parameter name: n -> number
// replaced sum(range(n + 1)) with a while loop to manually accumulate the sum
// introduced two new variables: totalSum and current